lang/gcc10/Portfile

```diff
@@ -96,16 +97,13 @@ configure.env-append \
                     OTOOL64=${prefix}/bin/otool
 
 # Make sure the MP clang version used by cctool's 'as' is available at
-# build time and used. Versions here must match those in cctools port.
-if       { ${os.major} == 12 || ${os.major} == 13 } {
-    depends_build-append port:clang-3.7
-    configure.compiler macports-clang-3.7
-} elseif { ${os.major} == 10 || ${os.major} == 11 } {
-    depends_build-append port:clang-3.4
-    configure.compiler macports-clang-3.4
-} elseif { ${os.major} >= 14 } {
-    depends_build-append port:clang-7.0
-    configure.compiler macports-clang-7.0
+# build time and used. Version list here must cover all possible variants
+# in the cctools port.
+foreach {llvm_v llvm_v_nodot} {3.4 34 3.7 37 3.9 39 4.0 40 5.0 50 6.0 60 7.0 70 8.0 80 devel dev} {
+    if {[active_variants cctools llvm${llvm_v_nodot} ""]} {
+        depends_build-append port:clang-${llvm_v}
+        configure.compiler macports-clang-${llvm_v}
```

Comment:
Can break here since the variants all conflict.

Reply:
I guess. I decided to just let the loop finish but I guess breaking is better...